qemu/tests/cfg/pktgen.cfg

```diff
@@ -2,6 +2,7 @@
     no JeOS
     no Windows
     no Host_RHEL.m5, Host_RHEL.m6
```

Comment:
`host_guest` this variant will test host <-> guest, so host should not a RHEL.8 host

Reply:
I have update it, please help me review it again. Thanks.